spec/controllers/two_factor_authentication/webauthn_verification_controller_spec.rb

```diff
@@ -291,6 +291,7 @@
             multi_factor_auth_method_created_at:
               second_webauthn_platform_configuration.created_at.strftime('%s%L'),
             webauthn_configuration_id: nil,
+            frontend_error: 'NotAllowedError',
```

Comment:
If this is being logged on the `Multi-Factor Authentication` event, maybe it should be documented in the [arguments](https://github.com/18F/identity-idp/blob/e958e9dfda40f1d5758dcf8c980aba257c004def/app/services/analytics_events.rb#L3112-L3144) for that method.

Though maybe that extra friction is enough to warrant persuing the [alternative](https://github.com/18F/identity-idp/pull/9611/files#r1396389193) instead.